Makefile

```diff
@@ -20,6 +20,8 @@ image-build:
 		--build-arg PKG=$(PKG) \
 		--build-arg SRC=$(SRC) \
 		--build-arg TAG=$(TAG) \
+		--build-arg MAJOR=$(shell ./scripts/version.sh ${TAG} major) \
+		--build-arg MINOR=$(shell ./scripts/version.sh ${TAG} minor) \
```

Comment:
looks like you've renamed the script, thank you, but missed these references to it